documentation/diagrams/distributed_diagram.gv

```diff
@@ -60,6 +60,15 @@ digraph distributed {
         fontcolor = red
     }
 
+    subgraph cluster_journey_fiflters {
+        Journey_Filters [label = "journey_filter", shape = "box", color = lightyellow, style = filled]
+
+        label = "(Common) Journey filtering"
```

Comment:
minor: I'd choose either `Common journey filtering` or `Journey filtering` (was not clear in previous remark)

Reply:
You have both here !